gtsfm/utils/viz.py

```diff
@@ -103,7 +110,7 @@ def plot_twoview_correspondences(
     corr_idxs_i1i2: np.ndarray,
     inlier_mask: Optional[np.ndarray] = None,
     dot_color: Optional[Tuple[int, int, int]] = None,
-    max_corrs: Optional[int] = 50,
+    max_corrs: Optional[int] = 5000,
```

Comment:
I think 5000 is too many, drawing so many is time consuming and makes the image cluttered. where have you found this useful?

Reply:
It's difficult to assess the quality of the matches by only drawing a small fraction of them.  I could expose it to the config if you'd like?